Library/Homebrew/diagnostic.rb

```diff
@@ -443,6 +443,29 @@ def check_access_cellar
         EOS
       end
 
+      def check_for_empty_taps
+        taps_dir = HOMEBREW_LIBRARY.join("Taps")
+        return unless taps_dir.exist?
+		return unless taps_dir.directory?
+
+        found_empty_dirs = []
+
+		# Delete all EMPTY taps_dir children directories
+        taps_dir.children.select{ |c| c.directory? }          \
```

Comment:
Minor nit, but I wouldn't bother too much about trying to wrap to ~80 for `doctor`-related stuff. It's pretty much a lost cause here given the required complexity of the checks & the diagnostic mechanism itself.